appveyor.yml

```diff
@@ -0,0 +1,52 @@
+version: 0.1.{build}
+
+cache:
+  - node_modules
+  - app\node_modules
+  - '%APPDATA%\npm-cache'
+  - '%USERPROFILE%\.electron'
+
+platform:
+  - x64
+
+init:
+  - git config --global core.autocrlf input
+  - set PATH=%GOROOT%\bin;%GOPATH%\bin;%PATH%
+
+install:
+  - ps: Install-Product node 6 x64
+  - git reset --hard HEAD
+  - npm install npm -g
+  - go version
+  - go env
+  - go get -u -v github.com/decred/dcrd
+  - go get -u -v github.com/decred/dcrwallet
+  - go get -u -v github.com/Masterminds/glide
```

Comment:
you can remove glide references if you have the dep there.

Reply:
But the glide is needed for `dcrd` isn't it?

Reply:
derp you're right.  someone is slow with it.... :)